listfilter.js

```diff
@@ -1,16 +1,105 @@
+                document.getElementById("2").innerHTML = "Id:"+obj1.Id+",name:"+obj1.Name+",color:"+obj1.color+",Price:"+obj1.Price+",company:"+obj1.Company+",Memory:"+obj1.Memory+",feature:camera:"+obj1.feature.camera+",touch:"+obj1.feature.touch;
+                document.getElementById("3").innerHTML = "Id:"+obj2.Id+",name:"+obj2.Name+",color:"+obj2.color+",Price:"+obj2.Price+",company:"+obj2.Company+",Memory:"+obj2.Memory+",feature:camera:"+obj2.feature.camera;
+                document.getElementById("4").innerHTML = "Id:"+obj3.Id+",name:"+obj3.Name+",color:"+obj3.color+",Price:"+obj3.Price+",company:"+obj3.Company+",Memory:"+obj3.Memory+",feature:camera:"+obj3.feature.camera+",sensor:"+obj3.feature.sensor+",touch:"+obj3.feature.touch;
+                document.getElementById("5").innerHTML = "Id:"+obj4.Id+",name:"+obj4.Name+",color:"+obj4.color+",Price:"+obj4.Price+",company:"+obj4.Company+",Memory:"+obj4.Memory+",feature:camera:"+obj4.feature.camera+",touch:"+obj4.feature.touch;
+                document.getElementById("6").innerHTML = "Id:"+obj5.Id+",name:"+obj5.Name+",color:"+obj5.color+",Price:"+obj5.Price+",company:"+obj5.Company+",Memory:"+obj5.Memory+",feature:camera:"+obj5.feature.camera+",sensor:"+obj5.feature.sensor+",touch:"+obj5.feature.touch;
+                document.getElementById("7").innerHTML = "Id:"+obj6.Id+",name:"+obj6.Name+",color:"+obj6.color+",Price:"+obj6.Price+",company:"+obj6.Company+",Memory:"+obj6.Memory+",feature:camera:"+obj6.feature.camera+",touch:"+obj6.feature.touch;
+                document.getElementById("8").innerHTML = "Id:"+obj7.Id+",name:"+obj7.Name+",color:"+obj7.color+",Price:"+obj7.Price+",company:"+obj7.Company+",Memory:"+obj7.Memory+",feature:camera:"+obj7.feature.camera+",sensor:"+obj7.feature.sensor+",touch:"+obj7.feature.touch;
+                document.getElementById("9").innerHTML = "Id:"+obj8.Id+",name:"+obj8.Name+",color:"+obj8.color+",Price:"+obj8.Price+",company:"+obj8.Company+",Memory:"+obj8.Memory+",feature:camera:"+obj8.feature.camera+",touch:"+obj8.feature.touch;
+                document.getElementById("10").innerHTML ="Id:"+obj9.Id+",name:"+obj9.Name+",color:"+obj9.color+",Price:"+obj9.Price+",company:"+obj9.Company+",Memory:"+obj9.Memory+",feature:camera:"+obj9.feature.camera;
+            });
+        } 
 
+    function showLaptopData(){
+        $("#l1").click(function(){
+            $("#hello12").show();
+            var ob=list2[0];
+            var ob1=list2[1];
+            var ob2=list2[2];
+            var ob3=list2[3];
+            var ob4=list2[4];
+            var ob5=list2[5];
+            var ob6=list2[6];
+            var ob7=list2[7];
+            var ob8=list2[8];
+            var ob9=list2[9];
+                document.getElementById("11").innerHTML = "Id:"+ob.Id+",name:"+ob.Name+",color:"+ob.color+",Price:"+ob.Price+",company:"+ob.Company+",Memory:"+ob.Memory+",feature:camera:"+ob.feature.camera+",sensor:"+ob.feature.sensor+",touch:"+ob.feature.touch;
```

Comment:
Put all this as well in same loop. [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself).